env0/resource_configuration_variable.go

```diff
@@ -110,6 +110,11 @@ func resourceConfigurationVariable() *schema.Resource {
 				Description: "the value of this variable must match provided regular expression (enforced only in env0 UI)",
 				Optional:    true,
 			},
+			"soft_delete": {
+				Type:        schema.TypeBool,
+				Description: "soft delete the configuration variable, once removed from the configuration it won't be deleted from env0",
+				Optional:    true,
```

Comment:
Please add a default value. This prevents many issues with bool types.

Reply:
what are the issues?